File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        st=""
        for x in s :
            st += str(ord(x)-96)
        while k :
            t=0
            for x in st :
                t += int(x)
            st=str(t)
            k -=1
        return t

File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.java
class Solution {
    public int getLucky(String s, int k) {
        int sum = 0;
        for (char ch : s.toCharArray()) {
            int a = ch - 96;
            sum += a % 10;     
            a /= 10;
            sum += a;  
        }
        k--;   
        while (k > 0) {
            int temp = sum;
            sum = 0;
            while (temp != 0) {
                sum += temp % 10;
                temp /= 10;
            }
            k--;
        }
        return sum;
    }
}